testament/categories.nim

```diff
@@ -517,7 +545,7 @@ proc isJoinableSpec(spec: TSpec): bool =
   result = not spec.sortoutput and
     spec.action == actionRun and
     not fileExists(spec.file.changeFileExt("cfg")) and
-    not fileExists(parentDir(spec.file) / "nim.cfg") and
```

Comment:
This is a heuristic and the tests all use ``nim.cfg``. Nothing to change here.

Reply:
pushed commit to address that, as described here https://github.com/nim-lang/Nim/pull/10044#discussion_r242873682